custom/icds_reports/templates/icds_reports/icds_app/mobile/navigation.directive.html

```diff
@@ -17,46 +17,11 @@
-                    <a href="{$ goToStep('maternal_and_child/stunting', $location.search()) $}" class="sub-links" ng-click="closeMenu()">
-                      Prevalence of Stunting (Height-for-Age)</a>
-                </div>
-                <div class="navigation-font pl-3 pr-3 mb-0" ng-class="{'active-mobile-link': $location.path().indexOf('/maternal_and_child/low_birth') !== -1}">
-                    <a href="{$ goToStep('maternal_and_child/low_birth', $location.search()) $}" class="sub-links" ng-click="closeMenu()">
-                      Newborns with Low Birth Weight</a>
-                </div>
-                <div class="navigation-font pl-3 pr-3 mb-0"
-                     ng-class="{'active-mobile-link': $location.path().indexOf('/maternal_and_child/early_initiation') !== -1}">
-                    <a href="{$ goToStep('maternal_and_child/early_initiation', $location.search()) $}" class="sub-links" ng-click="closeMenu()">
-                      Early Initiation of Breastfeeding</a>
-                </div>
-                <div class="navigation-font pl-3 pr-3 mb-0"
-                     ng-class="{'active-mobile-link': $location.path().indexOf('/maternal_and_child/exclusive_breastfeeding') !== -1}">
-                    <a href="{$ goToStep('maternal_and_child/exclusive_breastfeeding', $location.search()) $}" class="sub-links" ng-click="closeMenu()">
-                      Exclusive Breastfeeding</a>
-                </div>
-                <div class="navigation-font pl-3 pr-3 mb-0"
-                     ng-class="{'active-mobile-link': $location.path().indexOf('/maternal_and_child/children_initiated') !== -1}">
-                    <a href="{$ goToStep('maternal_and_child/children_initiated', $location.search()) $}" class="sub-links" ng-click="closeMenu()">
-                      Children initiated appropriate complementary feeding</a>
-                </div>
-                <div class="navigation-font pl-3 pr-3 mb-0"
-                     ng-class="{'active-mobile-link': $location.path().indexOf('/maternal_and_child/institutional_deliveries') !== -1}">
-                    <a href="{$ goToStep('maternal_and_child/institutional_deliveries', $location.search()) $}" class="sub-links" ng-click="closeMenu()">
-                      Institutional deliveries</a>
-                </div>
-                <div class="navigation-font pl-3 pr-3 mb-0"
-                     ng-class="{'active-mobile-link': $location.path().indexOf('/maternal_and_child/immunization_coverage') !== -1}">
-                    <a href="{$ goToStep('maternal_and_child/immunization_coverage', $location.search()) $}" class="sub-links" ng-click="closeMenu()">
-                      Immunization coverage (at age 1 year)</a>
+                <div ng-repeat="menuItem in navMenuItems['Maternal and Child Nutrition']"
```

Comment:
Is there a reason we have to break these out instead of doing the whole thing as a nested loop? I think you might have to change the data structure to a list of lists, but other than that not sure there's any other issues?

Reply:
Yeah. Initially I thought of doing it as a nested loop (that is where the unused children attribute came in). But the images (next to section headings) in web dashboard are material icons ('i' tags with different classes) like these.
`<i class="fa fa-lightbulb-o" aria-hidden="true"></i>`
`<i class="fa fa-users" aria-hidden="true"></i>`
Only If it were static images, I would have given image src in datastructure and looped through.

Reply:
what if the structure was like this:
```python
[
  {
    "name": "Maternal Child",
    "icon": "fa-users",
    "sub_pages": [
        // current structure
     ], 
  }
]

Reply:
> what if the structure was like this:
> 
> ```python
> [
>   {
>     "name": "Maternal Child",
>     "icon": "fa-users",
>     "sub_pages": [
>         // current structure
>      ], 
>   }
> ]
> ```

I will try using this